tests/virt_autotest/kubevirt_tests_server.pm

```diff
@@ -241,12 +291,17 @@ sub run_virt_tests {
     assert_script_run("mkdir -p $result_dir");
 
     # Run virt-tests command for each go test files
-    my $test_conf = 'kubevirt-tests.conf';
+    if (check_var('KUBEVIRT_TEST_SUITE', 'full')) {
+        $test_conf = 'full-tests.conf';
+        @kubevirt_tests = @full_tests;
+    } elsif (check_var('KUBEVIRT_TEST_SUITE', 'core')) {
```

Comment:
A separate core list is a workable way. I do not intend to suggest changing it. Just that I do not figure out why you do not directly use the way to customize test list by giving test_ids in ginkgo filter list, which is supported in existing automation already, right? Any reasons?

Reply:
Well, as the test ids were picked up from different test classes, I'd like to generate a test report for those test ids under matched test classes, so that we can know what the test comes from.

Reply:
I see. Make sense :)